query/src/sql/planner/binder/scalar.rs

```diff
@@ -160,6 +132,68 @@ impl ScalarBinder {
             ))),
         }
     }
+
+    fn bind_aggregate_op(
+        &self,
+        distinct: bool,
+        func_name: &Identifier,
+        args: &[Expr],
+        params: &[Literal],
+        bind_context: &BindContext,
+    ) -> Result<ScalarExprRef> {
+        let mut data_values = Vec::with_capacity(params.len());
+        for param in params.iter() {
+            data_values.push(match param {
+                Literal::Number(val) => DataValue::try_from_literal(val, None)?,
+                Literal::String(val) => DataValue::String(val.clone().into_bytes()),
+                Literal::Boolean(val) => DataValue::Boolean(*val),
+                Literal::Null => DataValue::Null,
+                Literal::Interval(_) => unimplemented!(),
+                Literal::CurrentTimestamp => unimplemented!(),
+            })
+        }
+
+        let scalar_binder = ScalarBinder::new();
+        let mut scalar_exprs = Vec::with_capacity(args.len());
+        for arg in args.iter() {
+            scalar_exprs.push(
+                scalar_binder
+                    .bind_expr(arg, bind_context)?
+                    .as_any()
+                    .downcast_ref::<Scalar>()
```

Comment:
We can use `enum-dispatch` instead of `downcast_ref`, see https://github.com/datafuselabs/databend/pull/5077
That's ok to do it in another PR :D

Reply:
I had tried in my local, but a blocking point is encountered due to https://github.com/datafuselabs/databend/blob/218821b4120e511ab9afacf25d0601a8ef90b340/query/src/sql/planner/binder/project.rs#L108

Reply:
Yes, I tries but failed.